src/OpenRiaServices.DomainServices.Server/Framework/OpenRiaServices.DomainServices.Server.csproj

```diff
@@ -1,14 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>net46;netcoreapp3.0</TargetFrameworks>
```

Comment:
Dont you need netstandard2.0 here aswell? or is it that net46 also supports netstandard2.0, so it's redundant? Since you are using it further down in the condition.

Reply:
Based on Entity Framework 6.3 not supporting netstandard 2.0 I am currently thinking about dropping netstandard 2.0 for the server and only targeting netcoreapp3.0 .

This means  VS 2019 version 16.3 will be required with netcore 3 workload installed so I will wait a little bit after the netcore 3.0 release and then try to make the changes